content/warnings/dont-call-proptypes.md

```diff
@@ -1,73 +1,72 @@
   highlighted: PropTypes.bool
 };
 ```
 
-Nothing changes here.
+Burada bir şey değişmiyor.
 
-### Don’t call PropTypes directly {#dont-call-proptypes-directly}
+### PropTypes'ı direkt olarak çağırmayın {#dont-call-proptypes-directly}
 
-Using PropTypes in any other way than annotating React components with them is no longer supported:
+PropTypes'ı React komponentlerini annotate ederek kullanmanın dışındaki yollar desteklenmiyor:
 
 ```javascript
 var apiShape = PropTypes.shape({
   body: PropTypes.object,
   statusCode: PropTypes.number.isRequired
 }).isRequired;
 
-// Not supported!
+// Desteklenmiyor!
 var error = apiShape(json, 'response');
 ```
+Eğer PropTypes'ı bu şekilde kullanma zorunluluğunuz varsa, size PropTypes'ın bir kopyasını oluşturmanızı öneririz([Bu](https://github.com/aackerman/PropTypes) [iki](https://github.com/developit/proptypes) paket gibi).
 
-If you depend on using PropTypes like this, we encourage you to use or create a fork of PropTypes (such as [these](https://github.com/aackerman/PropTypes) [two](https://github.com/developit/proptypes) packages).
+Eğer uyarıyı düzeltmezseniz, bu kod React 16 sürümüyle birlikte canlı ortamda çökecektir.
 
-If you don't fix the warning, this code will crash in production with React 16.
+### Eğer PropTypes'ı direkt çağırmadığınız halde uyarı alıyorsanız{#if-you-dont-call-proptypes-directly-but-still-get-the-warning}
 
-### If you don't call PropTypes directly but still get the warning {#if-you-dont-call-proptypes-directly-but-still-get-the-warning}
-
-Inspect the stack trace produced by the warning. You will find the component definition responsible for the PropTypes direct call. Most likely, the issue is due to third-party PropTypes that wrap React’s PropTypes, for example:
+Uyarıda belirtilen çalışma betiğini inceleyin. PropTypes'ı direkt olarak çağırmaya sebep olan komponenti bulacaksınız. Büyük ihtimalle uyarının sebebi React'in PropTypes özelliğini sarmallayarak kullanan bir 3. partidir, örneğin:
 
 ```js
 Button.propTypes = {
   highlighted: ThirdPartyPropTypes.deprecated(
     PropTypes.bool,
     'Use `active` prop instead'
   )
-}
+} 
 ```
 
-In this case, `ThirdPartyPropTypes.deprecated` is a wrapper calling `PropTypes.bool`. This pattern by itself is fine, but triggers a false positive because React thinks you are calling PropTypes directly. The next section explains how to fix this problem for a library implementing something like `ThirdPartyPropTypes`. If it's not a library you wrote, you can file an issue against it.
+Bu örnekte, `ThirdPartyPropTypes.deprecated` fonksiyonu `PropTypes.bool` sarmalayan bir fonksiyon. Bu kullanım kendi içerisinde uygun ancak React PropTypes'ı direkt çağırdınızı düşünerek yanlış pozitif olarak tetikler. Bir sonraki bölüm, `ThirdPartyPropTypes` gibi kütüphaneler kullandığınızda oluşan problemleri nasıl düzelteceğinizi açıklıyor. Eğer bu sizin yazdığınız bir kütüphane değilse, ilgili kütüphaneye sorun olarak bildirebilirsiniz.
```

Comment:
Suggestion. This might be better: 

> Bu örnekte, `ThirdPartyPropTypes.deprecated` fonksiyonu `PropTypes.bool`'u çağıran bir sarmalayıcıdır (wrapper).

Reply:
Fixed